package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Could you also check-in package-lock.json and fix for security alerts if there any, thanks

Reply:
fixed